python/tests/integration/connectors/memory/test_azure_cosmosdb_no_sql_memory_store.py

```diff
@@ -0,0 +1,196 @@
+# Copyright (c) Microsoft. All rights reserved.
+from typing import List
+
+import numpy as np
+import pytest
+from azure.cosmos import PartitionKey
+from azure.cosmos.aio import CosmosClient
+
+from semantic_kernel.memory.memory_record import MemoryRecord
+from semantic_kernel.memory.memory_store_base import MemoryStoreBase
+
+try:
+    from python.semantic_kernel.connectors.memory.azure_cosmosdb_no_sql.azure_cosmosdb_no_sql_memory_store import (
+        AzureCosmosDBNoSQLMemoryStore,
+    )
+
+    azure_cosmosdb_no_sql_memory_store_installed = True
+except AssertionError:
+    azure_cosmosdb_no_sql_memory_store_installed = False
```

Comment:
I don't see a `pytest.skip` or the equivalent if the library is not installed.

Reply:
Added